google/resource_resourcemanager_lien_test.go

```diff
@@ -0,0 +1,106 @@
+package google
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	resourceManager "google.golang.org/api/cloudresourcemanager/v1"
+)
+
+func TestAccResourceManagerLien_basic(t *testing.T) {
+	t.Parallel()
+
+	projectName := fmt.Sprintf("tf-test-%s", acctest.RandString(10))
+	org := getTestOrgFromEnv(t)
+	var lien resourceManager.Lien
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckResourceManagerLienDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccResourceManagerLien_basic(projectName, org),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckResourceManagerLienExists(
+						"google_resource_manager_lien.lien", projectName, &lien),
+				),
+			},
+			resource.TestStep{
+				ResourceName:      "google_resource_manager_lien.lien",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: func(_ *terraform.State) (string, error) {
```

Comment:
This has to be a function because otherwise lien.Name wouldn't resolve, right? Can you add a comment to that extent?

Reply:
Fixed.